src/app/contextBroker/contextBroker.cpp

```diff
@@ -1472,6 +1489,17 @@ int main(int argC, char* argV[])
   paParse(paArgs, argC, (char**) argV, 1, false);
   lmTimeFormat(0, (char*) "%Y-%m-%dT%H:%M:%S");
 
+  //
+  // FIXME P8: for release 0.24, -reqMutexPolicy != "all" ("all" is default) cannot
+  //           be used together with mongo subscription cache.
+  //           This limitation will be fixed in the next release (0.25.0)
+  //
+  if ((strcmp(reqMutexPolicy, "all") != 0) && (noCache == false))
+  {
+    LM_E(("Bad Input (reqMutexPolicy != 'all'  AND  subscription-cache CANNOT be used together in 0.24.0 (cache can be disabled using -noCache))"));
```

Comment:
New error text:

    LM_E(("This must be fixed for the 0.25.0 release!!!"));

Soon to be fixed